### PR TITLE
Fixes #139 added search for model listing 

### LINKED ISSUE #139: Model List: allow models to be searched by module
Currently, searching is only by name and version. 

There is no need for a search by any of the number columns - sort will be enough for these

### PR DESCRIPTION
**Pull request by rgemawat2000**
Fixed #139

### CODE REVIEW

File: web/onos-gui/src/app/onos-config/models-list/models-list.component.ts [hunk 1]
```diff
@@ -107,4 +110,11 @@ export class ModelsListComponent extends TableBaseImpl implements OnInit, OnDest
         }
         this.newConfigTitle = '';
     }
+
+    // onSort(colname: string) {
```
Comment (SeanCondon) [inline web/onos-gui/src/app/onos-config/models-list/models-list.component.ts:114]: this is probably not needed

**Comment by SeanCondon** [comment]
don't forget to add `Fixes #139` to the title of this PR

File: web/onos-gui/src/app/onos-config/models-list/models-list.component.ts (hunk 1 above)
Comment (rgemawat2000) [reply web/onos-gui/src/app/onos-config/models-list/models-list.component.ts:114]: these were there from before -  I will remove it

Comment (SeanCondon) [reply web/onos-gui/src/app/onos-config/models-list/models-list.component.ts]: I don't know - it showed up in green - thanks for removing

**Pull request merged**